spec/index.bs

```diff
@@ -513,6 +508,29 @@ interface mixin GPUObjectBase {
         An internal slot holding the [=device=] which owns the [=internal object=].
 </dl>
 
+{{GPUObjectDescriptorBase}} has the following members:
+
+<dl dfn-type=dict-member dfn-for=GPUObjectDescriptorBase>
+    : <dfn>label</dfn>
+    ::
+        The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
+</dl>
+
+
+: <dfn dfn>Debug label</dfn>
```

Comment:
optional nit: The `:`/`::` syntax is shorthand for `<dt>`/`<dd>` and inserts an implicit `<dl>` wrapper if there isn't one. So in this case this should render like a `<dl>`. Maybe it would be better if it just looks like a paragraph? (Otherwise I kind of feel like it needs something before it, describing what the list is, like "has the following members:" above.)